Dockerfile

```diff
@@ -3,3 +3,13 @@ WORKDIR /usr/src
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update -y
 RUN apt install -y curl cmake libgl1-mesa-dev libsdl2-dev libcurl4-openssl-dev libopenal-dev libfreetype6-dev mingw-w64 g++-mingw-w64 g++-multilib git zip vim-nox rsync
+COPY . /usr/src/
+
+RUN make -j
+
+ENTRYPOINT /usr/src/build/release-linux-x86_64/tremulous
+
+# run container with:
+#docker build -t tremulous:latest . && xhost +local:root && docker run --net=host -it -e DISPLAY=:0 -v /tmp/.X11-unix:/tmp/.X11-unix  --device /dev/snd tremulous:latest
+# see http://wiki.ros.org/docker/Tutorials/GUI for more info on running GUI applications with docker.
+# as running the game inside docker is heavily dependant on environment and requires questionable security practices, it is advised that one copies the built game out of the container through `docker cp`.
```

Comment:
Thanks for the note on "questionable security practices"- I wholely believe that the best usecase for a Tremulous runtime container is a dedicated "tremded" service that users can 
```bash
docker run tremulous_server -p 80083:80083 -v <local/game/mods/gpp24>:/whatever/game/mods/gpp24
```

Reply:
Done.